### PR TITLE
Reservations page

### LINKED ISSUE #24: [2pt]Create My Reservation page
create my reservation page

### PR DESCRIPTION
**Pull request by Mahamadou-Nouridine**
In this PR, I have done the following tasks:

- [ ] Create "My Reservations" page;
- [ ] Add house auto-filling when we click on the reserve button;
- [ ] Add house image URL checking when creating a new house

Screenshot 2023-10-03 211515

### CODE REVIEW

File: src/routes/Reservation.jsx
Comment (yashodhicy) [inline src/routes/Reservation.jsx:1]: [Optional] It is better to direct my reservation page after reserving a house. Now it stays in the same page.

File: src/components/Reservations/ReservationItem.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <i className="bi bi-calendar-event"></i>
+          </span>
+        </p>
+        <p>
+          <span className=" fw-bold">City: </span>{" "}
+          {reservation.house ? reservation.house.location : "...loading"}
+        </p>
+      </div>
+      <div className="d-flex flex-column align-items-center">
+        <h4 className="text-dark mb-4">Total sharge: ${reservation.total_charge}</h4>
```
Comment (yashodhicy) [inline src/components/Reservations/ReservationItem.jsx]: charge word mistakenly added as "Sharge".

**Review by yashodhicy — CHANGES_REQUESTED** [review]
#Change Requested.

You have done a nice job here and there is a minor issue you can fix that in no time.

File: src/components/Reservations/ReservationItem.jsx (hunk 1 above)
Comment (Mahamadou-Nouridine) [reply src/components/Reservations/ReservationItem.jsx]: Thanks

**Review by yashodhicy — APPROVED** [review]
# approved.
Welldone @Mahamadou-Nouridine . There is no other word than perfect for this. Congratulations on your awsome work.

**Comment by Mahamadou-Nouridine** [comment]
Thanks @yashodhicy

**Pull request merged**